sklearn/impute.py

```diff
@@ -426,11 +426,11 @@ def transform(self, X):
 class ChainedImputer(BaseEstimator, TransformerMixin):
     """Chained imputer transformer to impute missing values.
 
-    Basic implementation of chained imputer from MICE (Multivariate
-    Imputations by Chained Equations) package from R. This version assumes all
-    of the features are Gaussian.
+    Basic implementation of chained mutual regressions to find replacement
+    values in multivariate missing data. This version assumes all features
+    are Gaussian.
```

Comment:
Is it needed to mention this restriction of Gaussian? As I assume it depends on the predictor one uses.

Reply:
So far yes. The posterior sampling is done via a call to predict that also returns the standard deviation and then we sample from a Gaussian posterior. It's a limitation at the moment.